frontend/src/app/reducers/varianttests.js

```diff
@@ -38,9 +38,34 @@ const tests = [
 const chosenVariants = {};
 const identityReducers = {};
 
+let chosenTest = null;
+
 tests.forEach(test => {
-  chosenVariants[test.name] = test.getValue();
+  // Only put each user in one test. If we previously found a non-default test,
+  // put this user in default for the rest of the tests.
```

Comment:
That's correct, and I think the weights have to be chosen to make all the populations equal. That's why I chose to replace the white-border-test with the toggle-switch-on-homepage-test in my toggle switch pr.

Who can we ask about how to choose the weights properly when there are more tests?